cli/lsp/documents.rs

```diff
@@ -949,6 +949,22 @@ impl Documents {
     }
   }
 
+  /// Used by the tsc op_exists to shortcut trying to load a document to provide
+  /// information to CLI without allocating a document.
+  pub(crate) fn exists(&self, specifier: &ModuleSpecifier) -> bool {
```

Comment:
It's a little strange that we now have `exists` and `contains_specifier`, which both do the same thing. I don't think we need two methods. What's the performance impact of using `contains_specifier` vs `exist`? When TypeScript does an `exists` call it will then load the file, so is it a big deal to just load it when doing `.get()`?